oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/DocumentStoreIndexerBase.java

```diff
@@ -508,7 +507,7 @@ private void configureEstimators(IndexingProgressReporter progressReporter) {
     private long getEstimatedDocumentCount() {
         MongoConnection mongoConnection = indexHelper.getService(MongoConnection.class);
         if (mongoConnection != null) {
-            return mongoConnection.getDatabase().getCollection("nodes").count();
+            return mongoConnection.getDatabase().getCollection("nodes").countDocuments();
```

Comment:
```suggestion
            return mongoConnection.getDatabase().getCollection("nodes").estimatedDocumentCount();
```

`countDcuments` is very slow. I would use `estimatedDocumentCount` for faster results.

See https://www.mongodb.com/community/forums/t/countdocuments-is-extremly-slow/207357